tests/conftest.py

```diff
@@ -237,7 +237,8 @@ def run_remote_command(instances, command):
                 logger.info("Not able to print client deployment log")
 
             for service in [
-                "mender-client",
+                "mender-authd",
+                "mender-updated",
                 "mender-connect",
                 "mender-monitor",
                 "mender-gateway",
```

Comment:
This relates to my previous comment. If `integration` should be compatible with both versions of the recipe, then we should keep here `mender-client` -> So let's just clean-up `get_client_service_name` :broom: